platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -1620,6 +1620,14 @@ void DisplayServerX11::window_set_transient(WindowID p_window, WindowID p_parent
 			}
 		}
 	} else {
+		if (!wd_window.is_popup || !wd_window.no_focus) {
+			Atom type_atom = XInternAtom(x11_display, "_NET_WM_WINDOW_TYPE_DIALOG", False);
```

Comment:
It probably should also revert it to `_NET_WM_WINDOW_TYPE_NORMAL` when transient parent is unset?